tests/CSharpLib/CustomerId.cs

```diff
@@ -23,4 +26,17 @@ public static bool TryParse(string value,out CustomerId id)
             return false;
         }
     }
+#if NETSTANDARD2_1 
+    public interface ICustomerId{
```

Comment:
We don't have guidelines for C# in this projects, but at least could you leave a space after `;` and surround with spaces `{`, , `}` and `=`, which I think is kind of standard in both languages?

Reply:
As usual I'm a bit sloppy about indentation. Thanks!